dashboard/lib/routes/index.js

```diff
@@ -55,11 +47,17 @@ function getIndex(req, res) {
         logger.info({op: 'index#get'}, 'User info: %j', userinfo);
 
         if (userinfo !== undefined) {
-
             //search for subscription
 
             logger.debug({op: 'index#get'}, 'Regions: %s', regions.constructor.name);
 
+            if (regions.length === 0) {
+                res.render('error', {name: userinfo.displayName, role: req.session.role,
+                    message: 'The connection to Context Broker has timed out or the service is down',
```

Comment:
Why not the following message?

"Page Cannot be Displayed due to a Context Broker error, connection timed out or service was down.